BGL/include/CGAL/boost/graph/parameters_interface.h

```diff
@@ -64,6 +64,7 @@ CGAL_add_named_parameter(nb_points_per_distance_unit_t, nb_points_per_distance_u
 CGAL_add_named_parameter(outward_orientation_t, outward_orientation, outward_orientation)
 CGAL_add_named_parameter(overlap_test_t, overlap_test, do_overlap_test_of_bounded_sides)
 CGAL_add_named_parameter(preserve_genus_t, preserve_genus, preserve_genus)
+CGAL_add_named_parameter(do_stitch_per_cc_t, do_stitch_per_cc, do_stitch_per_cc)
```

Comment:
The named parameter could be more generic like `apply_per_connected_component` and could be used in other functions if needed

Reply:
Also should be tested